test/fuzz/yaml/yaml.go

```diff
@@ -20,10 +20,51 @@ limitations under the License.
 package yaml
 
 import (
+	"bytes"
+
 	"gopkg.in/yaml.v2"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	sigyaml "sigs.k8s.io/yaml"
 )
 
+// FuzzDuration is a fuzz target for unmarshaling Duration defined in "k8s.io/apimachinery/pkg/apis/meta/v1".
+// This target also checks that the unmarshaled result can be marshaled back to the input.
+func FuzzDuration(b []byte) int {
+	var unmarshalResult struct {
+		D metav1.Duration `json:"d"`
+	}
+	if err := yaml.Unmarshal(b, &unmarshalResult); err != nil {
+		return 0
+	}
+	marshalResult, err := sigyaml.Marshal(&unmarshalResult)
+	if err != nil {
+		panic(err)
+	}
+	if !bytes.Equal(marshalResult, b) {
+		panic("marshalResult != input")
+	}
+	return 1
+}
+
+// FuzzMicroTime is a fuzz target for unmarshaling MicroTime defined in "k8s.io/apimachinery/pkg/apis/meta/v1".
+// This target also checks that the unmarshaled result can be marshaled back to the input.
+func FuzzMicroTime(b []byte) int {
```

Comment:
adding a:
``` 
if len(b) < arbitraryShortNumber {
  return -1
}
```
To the top of these to make the target run much faster. I doubt we are going to find anything passed 256 bytes.

Reply:
Do you mean `if len(b) > 256`? Also, is this something that is worth adding to all of the targets in the file?

Reply:
Not sure yet... We can probably observe this for a bit and followup to optimize for iteration speed.

Reply:
Sounds good, I'll leave this as is for now then.